tests/monitoring/metrics.go

```diff
@@ -124,6 +124,40 @@ var _ = Describe("[sig-monitoring]Metrics", decorators.SigMonitoring, func() {
 			metrics := fetchPrometheusMetrics(virtClient, query)
 			Expect(metrics.Data.Result).To(BeEmpty(), "Expected no workqueue_depth metrics for virt workloads")
 		})
+
+		It("kubevirt workqueue metrics should include controllers names", func() {
+			names := []string{
+				"virt-operator",
+				"virt-handler-node-labeller",
+				"virt-handler-source",
+				"virt-handler-target",
+				"virt-handler-vm",
+				"virt-controller-disruption-budget",
+				"virt-controller-evacuation",
+				"virt-controller-export-vmexport",
+				"virt-controller-migration",
```

Comment:
Are we sure that previously we didn't get this name?

Reply:
@xpivarc reported to me that he couldn't find them. I ran everything and checked for workqueue metrics with "virt-controller-migration" name, and also couldn't see it

with my previous PR to use the init function for workqueue metrics, I mentioned I was able to see the migration controller impact on kubevirt_workqueue_adds_total, but it was not split by controller, that is, the value increased with migrations, but on a metrics without name = "virt-controller-migration"